README.md

```diff
@@ -78,8 +78,8 @@ These replaces should be removed prior to committing any updated module
 requires:
 
 ```
-$ go mod edit -dropreplace=github.com/decred/dcrd/rpc/jsonrpc/types/v2
-$ go mod edit -dropreplace=decred.org/dcrwallet/v2
+$ go mod edit -dropreplace=github.com/decred/dcrd/rpc/jsonrpc/types/v3
+$ go mod edit -dropreplace=decred.org/dcrwallet/v3
```

Comment:
this whole section would be better written encouraging the use of go.work

Reply:
Agreed.  I'll see about getting a separate PR up for that, but I think getting this updated to be accurate in the mean time makes sense.

Reply:
in that case, there is another replace example above this that you missed.

Reply:
Good call.  I noticed the dcrd types should've been /v4 as well.  Updated both.